www/new.spt

```diff
@@ -58,6 +57,7 @@ still_migrating = delta > 0
     <input type="hidden" name="csrf_token" value="{{ csrf_token }}">
 
     <label><h2>{{ _("Team Name") }}</h2></label>
+    <p><i>{{ _("Must contain alphabets. Can optionally contain numbers, dashes (-), underscores (_), periods (.), comma (,) and whitespaces.") }}</i></p>
```

Comment:
heh. It's petty [common](http://english.stackexchange.com/a/165611/106961) over here. I just realized myself that it's wrong though. :flushed: